src/main/java/com/securian/creditcompass/login/LoginController.java

```diff
@@ -1,22 +1,25 @@
 package com.securian.creditcompass.login;
 
-import com.securian.creditcompass.login.LoginService;
-import org.apache.coyote.Response;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.Map;
+
 @RestController
 public class LoginController {
-    private final LoginService loginService;
+    private final LoginInteractor loginService;
 
-    public LoginController(LoginService loginService) {
+    public LoginController(LoginInteractor loginService) {
 
         this.loginService = loginService;
     }
 
     @PostMapping("/login")
-    public ResponseEntity<Object> login(@RequestBody LoginDataTransferObject loginDataTransferObject) {
-        return loginService.authenticate(loginDataTransferObject.getUsername(), loginDataTransferObject.getPassword());
+    public ResponseEntity<Object> login(@RequestBody Map<String, String> userInput) {
+        String username = userInput.get("username");
+        String password = userInput.get("password");
+        LoginInputData loginInputData = new LoginInputData(username, password);
+        return loginService.authenticate(loginInputData);
```

Comment:
I like this change, much cleaner IMO and leads to less bugs on incoming request object resolution. Were there any resources that pointed to taking out the Login DTO from being a part of the request body? I think this should be ok from a CA standpoint, unless the DTOiszation should happen before the request even reaches the controller (confirm on this).